pyproject.toml

```diff
@@ -1,3 +1,34 @@
+[project]
+name = "torch"
+version = "2.8.0a0"
+description = "Tensors and Dynamic neural networks in Python with strong GPU acceleration"
+readme = "README.md"
+requires-python = ">=3.9"
```

Comment:
So, if we update ruff we can now remove the ruff specific Python min version, which is nice.